Waidhoferj/CSC-308-309-Project
# Braden50/applying style and adding errors

References #61 and #116

Got pycodestyle to pass for to all files in the backend
Created mutations.txt to store sample mutations instead of putting them at the bottom of mutations.py
Added docstrings to mutations and some api_types
Refactored some mutations to restrict changes we don't want
Added comments for future mutation refactoring

To test:

Did not really edit any functionality other than removing capabilities from mutations that we aren't using anyway. You could boot it up just to make sure it all works. Feel free to check out the mutation and type descriptions in the grapqhl interface if you're curious or want to critique my grammar. I wouldn't be hurt.
